src/pages/Checkout.tsx

```diff
@@ -246,23 +158,32 @@ export const Checkout = () => {
         >
           <Box
             sx={{
-              display: 'flex',
-              flexDirection: 'row',
-              alignItems: 'center',
-              gap: 1
+              display: 'inline-block'
             }}
           >
-            <Box>
-              <Typography
-                variant="h3"
-                sx={{
-                  marginTop: checkout.quote ? '-28px' : '0px'
-                }}
-              >
-                Your payment value is
-              </Typography>
-            </Box>
-            <PaymentCurrencySelector onChange={setPriceOverride} {...checkout} />
+            <Typography variant="h3">
+              Your payment value is&nbsp;
+              {formatPrice(
+                utils.parseEther(checkout.offer.price.public.toString()),
+                checkout.offer.price.currency
+              )}
+            </Typography>
+            {checkout.quote &&
+              checkout.quote.targetAmount &&
+              checkout.quote.targetCurrency && (
+                <Typography
+                  variant="h5"
+                  sx={{
```

Comment:
you can also do `textAlign: { xs: 'center', lg: 'right' }` 

That way you don't have to import the `useResponsive` hook

Reply:
nice improvement! thanks!